server/admin/member.js

```diff
@@ -99,6 +99,29 @@ function createLeaveHandler(ringpop) {
     };
 }
 
+function createReapHandler(ringpop) {
+    return function handleReap(arg1, arg2, hostInfo, callback) {
+        if (!ringpop.membership.localMember) {
+            process.nextTick(function() {
+                callback(errors.InvalidLocalMemberError());
+            });
+            return;
+        }
+
+        var reaped = 0;
+        ringpop.membership.members.forEach(function(member) {
+            if (member.status === Member.Status.faulty) {
+                reaped++;
+                ringpop.membership.makeTombstone(member.address, member.incarnationNumber);
+            }
+        });
+
+        process.nextTick(function() {
+            callback(null, null, JSON.stringify({'reaped': reaped}));
```

Comment:
I think we also need `'status': 'ok'` in here, to signal that the operation succeeded. The number of reaped nodes is only extra information.